src/js/module/API.js

```diff
@@ -0,0 +1,15 @@
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+
+const leaderBoard = () => {
+  fetch(baseURL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({ name: 'basketball' }),
+  })
+    .then((response) => response.json())
+    .then((json) => localStorage.setItem('gameId', JSON.stringify(json.result.split(' ')[3])))
+    .catch((error) => new Error('Error:', error));
+};
```

Comment:
## [REQUIRED]
### Please use `async` and `await` JavaScript features to consume the API as it's part of the project requirement. 👍